sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -453,7 +449,7 @@ def _get_bigquery_client(project: Optional[str] = None):
 {{ featureview.name }}__cleaned AS (
     SELECT base.*
     FROM {{ featureview.name }}__base as base
-    INNER JOIN {{ featureview.name }}__latest
+    INNER JOIN (SELECT * EXCEPT(row_number) FROM {{ featureview.name }}__latest WHERE row_number = 1)
```

Comment:
Any reason to do the `(SELECT * EXCEPT(row_number) FROM {{ featureview.name }}__latest WHERE row_number = 1)` here instead of doing `WHERE row_number = 1` when `__latest` is being created?